src/relayServerUtils.ts

```diff
@@ -270,6 +274,26 @@ async function validateIfTokenAmountIsAcceptable(
   }
 }
 
+type ERC20OptionalMethod = 'symbol' | 'decimals' | 'name';
+
+async function callERC20OptionalMethod<T>(
+  tokenInstance: ERC20,
+  methodName: ERC20OptionalMethod,
+  defaultValue: T
+): Promise<T> {
+  return tokenInstance[methodName]
```

Comment:
I would change this for a try and catch, since the `tokenInstance` is an ERC20 that comes from `rif-relay-contracts`, the method is gonna exists even if the smart contract does not have it. We can try and once fails setup the value or scrap the code searching for the 4 bytes.